packages/expo/src/ScreenOrientation/ExpoScreenOrientation.web.ts

```diff
@@ -1,8 +1,172 @@
+import { SyntheticPlatformEmitter } from 'expo-core';
+import { UnavailabilityError } from 'expo-errors';
+
+import { OrientationInfo, Orientation, OrientationLock } from './ScreenOrientation.types';
+
+enum OrientationLockType {
```

Comment:
```suggestion
enum WebOrientationLock {
```

"type" is already implied in all types unless describing the type of a type